app/components/ContextMenu/index.tsx

```diff
@@ -50,20 +52,35 @@ export default function ContextMenu({
   const previousVisible = usePrevious(rest.visible);
   const maxHeight = useMenuHeight(rest.visible, rest.unstable_disclosureRef);
   const backgroundRef = React.useRef<HTMLDivElement>(null);
+  const { ui } = useStores();
+  const setIsMenuOpen = useSidebar();
```

Comment:
```suggestion
  const setOpenMenuCount = useSidebar();
```

Need to name this better, perhaps.